gen-openapi.sh

```diff
@@ -8,3 +8,6 @@ docker run --rm \
   -g typescript-axios -i /local/swagger.yaml -o /local/src/openapi/
 
 rm swagger.yaml
+
+# prettier
+npm run lint:fix
```

Comment:
do not change script.
You should modify package.json

Reply:
like this
```
before
"gen:openapi": "bash ./gen-openapi.sh",
after
"gen:openapi": "bash ./gen-openapi.sh && npm run lint:fix",
```

Reply:
I fixed them.
Plz check, @mkXultra .